lbrynet/reflector/client/client.py

```diff
@@ -250,7 +250,7 @@ def send_descriptor_info(self):
     def skip_missing_blob(self, err, blob_hash):
         log.warning("Can't reflect blob %s", str(blob_hash)[:16])
```

Comment:
I changed it to only retry the upload once if it fails, and to include the traceback in the log